src/Components/CreateImageWizard/steps/Timezone/components/TimezoneDropDown.tsx

```diff
@@ -1,7 +1,152 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import {
+  Select,
+  SelectOption,
+  SelectList,
+  MenuToggle,
+  MenuToggleElement,
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+  Button,
+  FormGroup,
+} from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeTimezone,
+  selectTimezone,
+} from '../../../../../store/wizardSlice';
+
+const defaultTimezones = Intl.supportedValuesOf('timeZone');
```

Comment:
I was wondering how you would populate the list of timezones! I'm not sure this is the right approach though. If you look at the blueprint reference in our documentation, there is some guidance:

>The values supported by timezone can be listed by running the command:
>```bash
>$ timedatectl list-timezones
>```
https://osbuild.org/docs/user-guide/blueprint-reference#timezone

I cross referenced this with the output of Intl.suppertedValuesOf('timeZone') and they are in fact different!

Reply:
We might need to consult with some of the backend folks about this to be sure about the best way to do this.

The simplest option would be to just run `timedatectl` and put the output into an array we keep in `constants.ts`. But even then, could that be different for different versions of RHEL? Seems a bit excessive to add an endpoint for this to image-builder.

I've been thinking through how to figure this out ourselves and will just put some of my investigation here since the approach might be helpful for future customizations where we have questions like this.

One thing you can do to get a better idea about how these customizations are actually used during the build is to check the osbuild stages.

https://github.com/osbuild/osbuild/blob/f4b899873b1f108edbf12d81ee5dbed037238a7e/stages/org.osbuild.timezone

So here's how the timezone actually gets added in the osbuild stage:
```python
subprocess.run(["systemd-firstboot", f"--root={tree}", f"--timezone={zone}"], check=True)
```

If you run `man systemd-firstboot` there is some guidance:
![image](https://github.com/user-attachments/assets/a60f0d29-32dd-4f60-815b-0afa8de55f31)

Then if you do some additional research, you'll realize that `timedatectl` is installed alongside `systemd`. So it makes sense that we use `timedatectl` as the source of truth for the timezones.

Reply:
Quick example of how the two outputs are different - `timedatectl list-timezones` has a 'Zulu' timezone. Intl.supportedValuesOf('timeZone') does not.

Reply:
Oh no... started a thread on the main channel. Looks like the lists change based on release